src/OpenTelemetry.Exporter.ZPages/OpenTelemetry.Exporter.ZPages.csproj

```diff
@@ -6,6 +6,10 @@
     <PackageTags>$(PackageTags);ZPages;distributed-tracing</PackageTags>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Not sure this needed - new csproj format automatically includes all files.

Reply:
yeah, I'm surprised VS added this. @n1dtec can you please try to remove it?